zigzag/zigzag_test_log.py

```diff
@@ -529,6 +542,21 @@ def _lookup_ids(self):
         except IndexError:  # test case has not been created yet in qTest
             pass
 
+    def _lookup_job_config_attributes(self):
+        """ Finds the array of job config attributes on the end of the job name.
+
+        Returns:
+            List: of job config attributes.
+        """
+        logs = self._mediator.junit_xml.findall('testcase')
```

Comment:
Given that the bracket information is per test case then the `self._testcase_xml` attribute has all the information that is necessary to gather the bracket information.